src/EditorFeatures/Core/Implementation/Organizing/OrganizeDocumentCommandHandler.cs

```diff
@@ -56,6 +57,11 @@ public bool ExecuteCommand(OrganizeDocumentCommandArgs args, CommandExecutionCon
             return true;
         }
 
+        public VSCommanding.CommandState GetCommandState(SortImportsCommandArgs args)
+        {
+            return GetCommandState(args, o => o.SortImportsDisplayStringWithAccelerator);
```

Comment:
Does GetCommandState need an update to handle the fact this could be done syntactic only?